usercommandmgmt/usercommandmgmt.py

```diff
@@ -0,0 +1,226 @@
+        else:
+            if self.activeDb.BelongsToUser(command, ctx.message.author.id) == False:
+                await ctx.send("Hey, that's not yours.")
+                return
+
+            try:
+                await self.commandobj.delete(ctx=ctx, command=command)
+                await self.activeDb.DeleteFromDb(command)
+                await ctx.send("Custom command successfully deleted.")
+            except NotFound:
+                await ctx.send("That command doesn't exist.")
+
+    @customcom.command(name="edit")
+    async def cc_edit(self, ctx, command: str.lower, *, text: str = None):
+        """Edit a custom command.
+
+        Example:
+            - `[p]customcom edit yourcommand Text you want`
+
+        **Arguments:**
+
+        - `<command>` The custom command to edit.
+        - `<text>` The new text to return when executing the command.
+        """
+        if ctx.message.author.top_role.permissions.administrator:
+            await ctx.send("Admin Override.")
+
+            try:
+                await self.commandobj.edit(ctx=ctx, command=command, response=text)
+                await ctx.send("Custom command successfully edited.")
+            except NotFound:
+                await ctx.send(
+                    (
+                        "That command doesn't exist. Use"
+                        + ctx.clean_prefix
+                        + " to add it."
+                    )
+                )
+            except ArgParseError as e:
+                await ctx.send(e.args[0])
+            except CommandNotEdited:
+                pass
+        else:
+            # blocks a user from editing a command that isnt theirs
+            if self.activeDb.BelongsToUser(command, ctx.message.author.id) == False:
+                await ctx.send("Hey, that's not yours.")
+                return
+
+            try:
+                await self.commandobj.edit(ctx=ctx, command=command, response=text)
+                await ctx.send("Custom command successfully edited.")
+            except NotFound:
+                await ctx.send(
+                    (
+                        "That command doesn't exist. Use"
+                        + ctx.clean_prefix
+                        + " to add it."
+                    )
+                )
+            except ArgParseError as e:
+                await ctx.send(e.args[0])
+            except CommandNotEdited:
+                pass
+
+    def GetHighestUserCommAllowance(self, member):
+        """
+        returns the highest number of custom commands permitted by the user's roles
+        """
+        # all of the given user's roles
+        usr_roles = member.roles
+        # the user's roles that confer different command allowances
+        rel_usr_roles = [0]
+
+        for cmd in usr_roles:
+            allowance = role_cmd_limits.get(cmd.name, 0)
+            if allowance != 0:
+                rel_usr_roles.append(allowance)
+
+        return max(rel_usr_roles)
```

Comment:
You're not using `self` here, perhaps make this a static function? The way you are using it in the other helper is like that.